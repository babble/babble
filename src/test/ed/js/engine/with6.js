
a = { z : 5 };
with( a ) print( z );

b = { z : 6 };
with( a ){
    print( z );
    with( b ){
        print( z );
    }
    print( z );
}


a = { z : 1 };
b = null;
print( "1 = " + a. z );

with ( a ){

    print( "1 = " + z );
    z = 4;
    print( "4 = " + z );

    b = 3;
    print( "3 = " + b );
}

print( "*4 = " + a.z );
print( "null = null " );
print( "3 = " + b );
