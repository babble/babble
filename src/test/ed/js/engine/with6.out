5
5
6
5
1 = 1
1 = 1
4 = 4
3 = 3
*4 = 4
null = null 
3 = 3
